app/build.gradle

```diff
@@ -15,6 +18,8 @@ android {
         versionName "9.6.2"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        resValue "string", "PASS_FOR_ZIP", properties.getProperty("PASS.FOR.ZIP", "")
```

Comment:
use the current approach of the project for getting local.properties data, like the weatherAPIkey.